src/riskmatrix/views/risk_assessment.py

```diff
@@ -154,14 +156,37 @@ def buttons(
         return assessment_buttons(assessment, self.request)
 
 
-_RADIO_TEMPLATE = Markup("""
+class AssessmentOverviewTable(AssessmentBaseTable):
+    nr = DataColumn(_("Nr."))
+    name = DataColumn(_("Name"))
+    description = DataColumn(_("Description"), class_name="visually-hidden")
+    category = DataColumn(_("Category"))
+    asset_name = DataColumn(_("Asset"))
+    likelihood = DataColumn(_("Likelihood"))
+    impact = DataColumn(_("Impact"))
+
+    def __init__(self, org: "Organization", request: "IRequest") -> None:
+        super().__init__(org, request, id="risks-table")
+        xhr_edit_js.need()
+
+    def query(self) -> "Generator[RiskMatrixAssessment]":
```

Comment:
```suggestion
    nr = DataColumn(_('Nr.'))
    name = DataColumn(_('Name'))
    description = DataColumn(_('Description'), class_name='visually-hidden')
    category = DataColumn(_('Category'))
    asset_name = DataColumn(_('Asset'))
    likelihood = DataColumn(_('Likelihood'))
    impact = DataColumn(_('Impact'))

    def __init__(self, org: 'Organization', request: 'IRequest') -> None:
        super().__init__(org, request, id='risks-table')
        xhr_edit_js.need()

    def query(self) -> 'Iterator[RiskMatrixAssessment]':
```
Just a small consistency nit, use single quotes for strings, so we don't mix styles.